src/rich-text/commands/index.ts

```diff
@@ -421,6 +474,43 @@ const headingDropdown = (schema: Schema) =>
         )
     );
 
+const overflowDropdown = (schema: Schema, options: CommonViewOptions) =>
+    makeMenuDropdown(
+        "Overflow",
+        _t("commands.overflow"),
+        "overflow-dropdown",
+        () => true,
+        () => false,
```

Comment:
Currently, this causes the dropdown to never be set as active, even when the cursor is otherwise inside an active child.

Reply:
On second thought, don't worry about this. I think I'll extend the menu helper to do this automatically.